### ISSUE TITLE
Feature: Restarting tabs should close the old tab, and put the new tab in the same place as the original.

### ISSUE THREAD

**Issue by tomcorke** [tomcorke/obsidian-work-terminal#83]
When using tab context menu "restart' option the old tab should be closed.

The new tab should be placed in the tab order where the old one was - so the user may continue uninterupted.
